cmd/pluginctl/cmd/build.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "build PLUGIN_DIR",
+		Short: "Build plugin from directory",
+		Args:  cobra.ExactArgs(1),
+	}
+
+	flags := cmd.Flags()
+	tag := flags.String("t", "", "image tag")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		path, err := filepath.Abs(args[0])
+		if err != nil {
+			return fmt.Errorf("invalid path: %s", err.Error())
+		}
+
+		name := filepath.Base(path)
+
+		if *tag == "" {
+			*tag = fmt.Sprintf("10.31.81.1:5000/waggle/%s", name)
```

Comment:
do we want to assume `waggle` here?

i know Nico was building stuff as: `panda/image` (or something)